web-model/src/main/java/com/oasis/webmodel/response/LoginResponseModelFAIL.java

```diff
@@ -0,0 +1,14 @@
+package com.oasis.webmodel.response;
+
+import lombok.Getter;
+
+@Getter
+public class LoginResponseModelFAIL {
```

Comment:
no all capital case,
change it to `LoginFailedResponse`

Reply:
Commit ID: 279a1f3